File: NetFramework/TreeView/VB/BoundModeWithBindingList/AddNewColumnForm.vb
Public Class AddNewColumnForm

    Public ReadOnly Property ColumnName As String
        Get
            Return tbName.Text
        End Get
    End Property

    Public ReadOnly Property ColumnHeaderText As String
        Get
            Return tbHeaderText.Text
        End Get
    End Property

    Public ReadOnly Property ColumnDisplayFieldName As String
        Get
            Return tbDisplayFieldName.Text
        End Get
    End Property

End Class

File: NetFramework/TreeView/CS/BoundModeWithBindingList/AddNewColumnForm.cs
using System.Windows.Forms;

namespace BoundModeWithBindingList
{
    public partial class AddNewColumnForm : Form
    {
        public AddNewColumnForm()
        {
            InitializeComponent();
        }

        public string ColumnName
        {
            get { return tbName.Text; }
        }

        public string ColumnHeaderText
        {
            get { return tbHeaderText.Text; }
        }

        public string ColumnDisplayFieldName
        {
            get { return tbDisplayFieldName.Text; }
        }
    }
}
